glide.lock

```diff
@@ -1,6 +1,15 @@
-hash: d6512ee48265f77a1c34516c5ffb73828ccb937533a56454568e6ed2211ba607
-updated: 2017-09-20T11:55:50.9196227-07:00
+hash: 86524339d17f3edec1d59ef1349c00a011ffbd51245f8c3b10b6967c76c513a9
+updated: 2017-10-01T18:35:39.603933045-07:00
 imports:
+- name: cloud.google.com/go
```

Comment:
There is so much random stuff in the glide.lock ; no idea why. 

Can you just delete everything except the thriftrw version bump ?

Reply:
AFAIK, `glide update` always updates the entire glide.lock file.

Reply:
yeah glide is dumb so we should hand edit the glide.lock to undo the bugs in glide :(

Reply:
Well glide update also updates the hash at the top of the glide.lock file.  If we hand edit this file, we might break that hash!

Reply:
i dont think the hash matters XD